includes/Renderer/Stories/Renderer.php

```diff
@@ -567,17 +568,31 @@ protected function render_story_with_poster() {
 						alt="<?php echo esc_attr( $story->get_title() ); ?>"
 					>
 					</amp-img>
-				<?php } else { ?>
-					<img
-						src="<?php echo esc_url( $poster_url ); ?>"
-						alt="<?php echo esc_attr( $story->get_title() ); ?>"
-						width="<?php echo absint( $this->width ); ?>"
-						height="<?php echo absint( $this->height ); ?>"
-					>
-				<?php } ?>
+					<?php
+				} else {
+
+					printf(
+						'<img src="%1$s" width="%2$d" height="%3$d" alt="%4$s" class="%5$s" />',
+						esc_url( $story->get_poster_portrait() ),
+						absint( $this->width ),
+						absint( $this->height ),
+						esc_attr( $story->get_title() ),
+						esc_attr( 'wp-image-' . $story->get_thumbnail_id() )
+					);
```

Comment:
IMO it was more readable before. Plus, we don't use `printf` for `amp-img` above.

Can we use the  previous way of writing `<img>` again for consistency?